testdata/7/code.go

```diff
@@ -0,0 +1,31 @@
+package code
+
+import (
+	"errors"
+)
+
+// Those are not errors
+var myVar = 1
+
+// var errFakeErrorUnexported = 1
+// var ErrFakeErrorExported = 1
```

Comment:
I think you should include these. Even though we know they aren't errors, including them captures the existing behavior, and that's valuable for when we improve this in the future since the change in behavior will also be captured then.

Reply:
Done.